lib/ansible/plugins/callback/default.py

```diff
@@ -115,6 +115,9 @@ def v2_on_file_diff(self, result):
         if 'diff' in result._result and result._result['diff']:
             self._display.display(self._get_diff(result._result['diff']))
 
+    def _get_censored_item(self, result):
```

Comment:
I considered that- depends on if we want other callback plugins to have the ability to censor at will, or keep that as a choice made by the infrastructure... IIRC, we're not always deepcopying the inputs to the callback plugins, so censoring the data in-place may have unintended consequences. If we're sure that all the different paths are deep-copied, though, I can probably move it elsewhere.